### ISSUE TITLE
Global: Sometimes game gets stuck in loop and keeps refreshing main group screen. Has to do with player's screens in some way. 

### ISSUE THREAD

**Issue by jpailthorpe** [engagementgamelab/emerging-citizens#189]
Similar and related to  #179 

Please see video for reference. Was playing HTYI and after about 31 seconds, continually, the game would "refresh" and player screens would refresh, showing the same screen but restarting the timer. This happened over and over. 

[https://drive.google.com/open?id=0B2iJJeD7RJtgS0YtSkNfUUdJc0k](url)

**Comment by jpailthorpe** [comment]
update on this: we found that this would happen in wikigeeks around the same time EVERYTIME. this happened at :58-59 seconds.

**Comment by jpailthorpe** [comment]
confirming that I was abl to recreate this on HTYI again today:

1. four players in game. category science and technology. 
2. all three players were functioning correctly, but one player's screen never moved on automatically as the game progressed.
3. played through one round after refreshing the faulting player screen and was able to play normally.
4. in round 3, the game started through a refresh cycle seen similarly in above video + in issues #179 #214 #190 . It ALWAYS happens at the same time (about halfway, at 30 seconds). 
5. I refreshed the player's screen that kept reloading in sync with group screen, and fired off tweets across all players devices before the timer could refresh on me. 
6. once I did that, the game functioned again normally.
